docs/src/userguide/language_basics.rst

```diff
@@ -655,8 +655,8 @@ error return value.
 While this is always the case for Python functions, functions
 defined as C functions or ``cpdef``/``@ccall`` functions can return arbitrary C types,
 which do not have such a well-defined error return value.
-Extra care must be taken to ensure Python exceptions are correctly
-propagated from such functions.
+Cython by default uses dedicated value to propagate exceptions from a non-external ``cpdef``/``@ccall``
+functions. Moreover, the way how an exception is propagated can be changed if needed.
```

Comment:
```suggestion
By default Cython uses a dedicated return value to signal that an exception has occurred in non-external ``cpdef``/``@ccall``
functions. However, how Cython handles exceptions from these functions can be changed if needed.
```
Small re-write...

Reply:
:-/ wrong branch. I have another PR for docs. Should I move it there or can it stay here?